CHANGELOG.md

```diff
@@ -220,6 +220,7 @@ _📚 Refer to the [GitHub Release Notes](https://github.com/munich-quantum-tool
 
 <!-- PR links -->
 
+[#1271]: https://github.com/munich-quantum-toolkit/core/pull/1271
```

Comment:
@MatthiasReumann would you mind fixing the order of the links here?
Some of the latest ones seem out of order to me.